src/tarantool-gdb.py

```diff
@@ -0,0 +1,468 @@
+
+    def entry_type(self):
+        return self.__entry_type
+
+    def entry_from_item(self, item):
+        return (item.cast(gdb.lookup_type('char').pointer()) - self.__offset).cast(self.__entry_type.pointer()).dereference()
+
+def build_list_variables_map():
+    list_variables = [
+        ( 'box_on_call', 'trigger::link' ),
+        ( 'box_on_select', 'trigger::link' ),
+        ( 'box_on_shutdown_trigger_list', 'trigger::link' ),
+        ( 'box_raft_on_broadcast', 'trigger::link' ),
+        ( 'engines', 'engine::link' ),
+        ( 'log_rotate_list', 'log::in_log_list' ), #static
+        ( 'on_access_denied', 'trigger::link' ),
+        ( 'on_alter_func', 'trigger::link' ),
+        ( 'on_alter_sequence', 'trigger::link' ),
+        ( 'on_alter_space', 'trigger::link' ),
+        ( 'on_console_eval', 'trigger::link' ),
+        ( 'on_schema_init', 'trigger::link' ),
+        ( 'on_shutdown_trigger_list', 'on_shutdown_trigger::link' ),
+        ( 'popen_head', 'popen_handle::list' ),
+        ( 'replicaset_on_quorum_gain', 'trigger::link' ),
+        ( 'replicaset_on_quorum_loss', 'trigger::link' ),
+        ( 'session_on_auth', 'trigger::link' ),
+        ( 'session_on_connect', 'trigger::link' ),
+        ( 'session_on_disconnect', 'trigger::link' ),
+        ( 'shutdown_list', 'session::in_shutdown_list' ),
+    ]
+    ret = {}
+    for variable, entry in list_variables:
+        try:
+            ret[variable] = TtListEntryInfo(entry)
+        except Exception as exc:
+            gdb.write('{}\n'.format(str(exc)))
+    return ret
+
+def build_list_containers_map():
+    list_containers = [
+        ( 'alter_space::ops', 'AlterSpaceOp::link' ),
+        ( 'alter_space::key_list', 'index_def::link' ),
+        ( 'applier::on_state', 'trigger::link' ),
+        ( 'cbus::endpoints', 'cbus_endpoint::in_cbus' ),
+        ( 'checkpoint::entries', 'checkpoint_entry::link' ),
+        ( 'cord::alive', 'fiber::link' ),
+        ( 'cord::dead', 'fiber::link' ),
+        ( 'cord::ready', 'fiber::state' ),
+        ( 'cpipe::on_flush', 'trigger::link' ),
+        ( 'create_ck_constraint_parse_def::checks', 'ck_constraint_parse::link' ),
+        ( 'create_fk_constraint_parse_def::fkeys', 'fk_constraint_parse::link' ),
+        ( 'fiber::on_stop', 'trigger::link' ),
+        ( 'fiber::on_yield', 'trigger::link' ),
+        ( 'fiber::wake', 'fiber::state' ),
+        ( 'fiber_channel::waiters', 'fiber::state' ),
+        ( 'fiber_cond::waiters', 'fiber::state' ),
+        ( 'fiber_pool::idle', 'fiber::state' ),
+        ( 'func::func_cache_pin_list', 'func_cache_holder::link' ),
+        ( 'gc_checkpoint::refs', 'gc_checkpoint_ref::in_refs' ),
+        ( 'gc_state::checkpoints', 'gc_checkpoint::in_checkpoints' ),
+        ( 'index::full_scans', 'full_scan_item::in_full_scans' ),
+        ( 'index::nearby_gaps', 'gap_item::in_nearby_gaps' ),
+        ( 'iproto_thread::stopped_connections', 'iproto_connection::in_stop_list' ),
```

Comment:
just a little bit :)
I consider 80 symbols limit only as recommendation, not as a strict rule. Here we have a lot of items, one per line, so I assume it's not a big deal if a few of them cross 80 sym boundary. I think visually it's more consistent than it would be if some items occupy 2 line while others occupy 1.

I would keep current approach